package.json

```diff
@@ -11,30 +11,31 @@
     "dashboard": "vue ui"
   },
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "^1.2.15",
-    "@fortawesome/free-brands-svg-icons": "^5.7.2",
-    "@fortawesome/free-regular-svg-icons": "^5.7.2",
-    "@fortawesome/free-solid-svg-icons": "^5.7.2",
-    "@fortawesome/vue-fontawesome": "^0.1.5",
-    "bootstrap-vue": "^2.0.0-rc.12",
-    "vue": "^2.5.22",
-    "vue-router": "^3.0.1",
+    "@fortawesome/fontawesome-svg-core": "^1.2.17",
+    "@fortawesome/free-brands-svg-icons": "^5.8.1",
+    "@fortawesome/free-regular-svg-icons": "^5.8.1",
+    "@fortawesome/free-solid-svg-icons": "^5.8.1",
+    "@fortawesome/vue-fontawesome": "^0.1.6",
+    "@hapi/hoek": "^6.2.1",
```

Comment:
I get the other updates, but why is hoek added here?

Reply:
I saw a warning during npm install that said it was now maintained at hapi/hoek.